Source/Blazorise/Base/BaseAfterRenderComponent.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+
+namespace Blazorise.Base
+{
+    public abstract class BaseAfterRenderComponent : ComponentBase, IDisposable

```

Comment:
I don't really like separating RenderQueue into another base component. Can you revert it to BaseComponent?

Reply:
Sure if that's what you prefer. Would you like that I make it into a "feature" class and inject it where it's needed (DatagridBase and BaseComponent), or would you prefer that I copy paste the code instead and we maintain in two places for now.

And may I ask why, though? :P

Reply:
I was thinking in the meantime and maybe it can stay as a separate base class. I didn't notice at first that you also inherited from it on the DataGrid component. So let it be. Just format it accordingly to the codebase.

Reply:
Alright then.
Do you use codemaid or something to auto format with the regions? If so, could you share the settings?

Reply:
Well... done.

Reply:
Sorry I was out yesterday so I didn't have any time left to review. Gonna now!